main.go

```diff
@@ -109,26 +110,22 @@ func localSetup(localPath string, config sequinsConfig) *sequins {
 }
 
 func s3Setup(bucketName string, path string, config sequinsConfig) *sequins {
-	auth, err := aws.GetAuth(config.S3.AccessKeyId, config.S3.SecretAccessKey, "", time.Time{})
-	if err != nil {
-		log.Fatal(err)
-	}
-
 	regionName := config.S3.Region
-	if regionName == "" {
-		regionName = aws.InstanceRegion()
-		if regionName == "" {
-			log.Fatal("Unspecified S3 region, and no instance region found.")
-		}
-	}
 
-	region, exists := aws.Regions[regionName]
-	if !exists {
-		log.Fatalf("Invalid AWS region: %s", regionName)
-	}
+	// Requiring region name in config for now
+	// if regionName == "" {
+	// 	regionName = aws.InstanceRegion()
+	// 	if regionName == "" {
+	// 		log.Fatal("Unspecified S3 region, and no instance region found.")
+	// 	}
+	// }
```

Comment:
Can we default to the instance region, still? I'm not sure without looking but I think this'll break our deployment on ec2.